substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -379,6 +379,17 @@ pub mod pallet {
 	pub type Nominators<T: Config> =
 		CountedStorageMap<_, Twox64Concat, T::AccountId, Nominations<T>>;
 
+	/// Stakers whose funds are managed by other pallets.
+	///
+	/// This pallet does not apply any locks on them, therefore they are only virtually bonded. They
+	/// are expected to be keyless accounts and hence should not be allowed to mutate their ledger
+	/// directly via this pallet. Instead, these accounts are managed by other pallets and accessed
+	/// via low level apis. We keep track of them to do minimal integrity checks.
+	// TODO(ank4n): Can we keep this entry in `Ledger`? Or migrate later in conjunction with
```

Comment:
Since this PR will keep the metadata of whether a ledger is virtual or not in storage, wdyt about opening an (unconfirmed) issue to keep track of this TODO and link it here?

Reply:
https://github.com/paritytech/polkadot-sdk/issues/4217